contrib/gitea-pull-push

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+user=$(echo $1 | cut -d ':' -f 1)
```

Comment:
```suggestion
# $1 = user:branch as per copy button
# $2 = repo name
user=$(echo $1 | cut -d ':' -f 1)
```
And whatever $3 and $4 are?

Reply:
well just of you wana add extra stuff like `-f` or so

Reply:
But why exactly two arguments?
Why not shift twice and use `$@` instead?